ios/brave-ios/Sources/BraveWallet/Crypto/Transactions/TransactionParser.swift

```diff
@@ -138,41 +139,41 @@ enum TransactionParser {
   ) -> ParsedTransaction? {
     let fromAccountInfo =
       accountInfos.first(where: { $0.accountId == transaction.fromAccountId }) ?? .init()
-    let formatter = WalletAmountFormatter(
-      decimalFormatStyle: decimalFormatStyle ?? .decimals(precision: Int(network.decimals))
-    )
```

Comment:
Why did this move to be defined in each `TransactionType`? Seems unnecessary code duplication, we're still using the same initialization in each type?